tests/test_util.py

```diff
@@ -78,7 +81,7 @@ def test_minimize_smooth():
 
 
 def test_minimize_array():
-    assert _round_eq(2, ds.minimize(lambda x: (x[0]-2)**2, [0], array=True))
+    assert _round_eq(2, ds.minimize(lambda x: (x[0]-2)**2, [0], array=True, log=print))
```

Comment:
Can you help me understand the purpose of this change?

Reply:
+1